### PR TITLE
Duplicate "the" in jsp files Fixes #6460

### LINKED ISSUE #6460: Duplicate "the" in jsp files
Remove the 3 extra `the`s from
1. Comment in `<form id="ajaxLoaderDataForm">` in `adminEmailLog.jsp`
2. The second `<li>` under `Enroll students by doing one of the following options` in `instructorHelp.jsp`
3. The first `<span>` after `Tolerate unequal contributions` in `instructorHelp.jsp`

### PR DESCRIPTION
**Pull request by iamhunter**
Removed the 3 extra thes from

Comment in <form id="ajaxLoaderDataForm"> in adminEmailLog.jsp
The second <li> under Enroll students by doing one of the following options in instructorHelp.jsp
The first <span> after Tolerate unequal contributions in instructorHelp.jsp

This is my first pull request hopefully I won’t screw something up

### CODE REVIEW

File: src/main/webapp/instructorHelp.jsp [hunk 1]
```diff
@@ -4775,7 +4775,7 @@
                                                 </ul>
                                             </div>
                                             <div class="feedback-path-others margin-top-7px" style="display:none;">
-                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">  
+                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: and here

File: src/main/webapp/instructorHelp.jsp [hunk 2]
```diff
@@ -5373,7 +5373,7 @@
                                               </ul>
                                           </div>
                                           <div class="feedback-path-others margin-top-7px" style="display:none;">
-                                              <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">  
+                                              <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: and here

File: src/main/webapp/instructorHelp.jsp [hunk 3]
```diff
@@ -5973,7 +5973,7 @@
...
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: and here

File: src/main/webapp/instructorHelp.jsp [hunk 4]
```diff
@@ -6337,7 +6337,7 @@
                                                 </ul>
                                             </div>
                                             <div class="feedback-path-others margin-top-7px" style="display:none;">
-                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">  
+                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: and here

File: src/main/webapp/instructorHelp.jsp [hunk 5]
```diff
@@ -7603,7 +7603,7 @@
...
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: and here

File: src/main/webapp/instructorHelp.jsp [hunk 6]
```diff
@@ -7913,16 +7913,16 @@
...
                 </div>
-                
+
                 <div class="row">
                     <div class="col-sm-12">
                         <a name="fbRankOptions">
                             <h4>Rank Options question</h4>
                         </a>
-                        Rank options questions are question where the students rank options that are created by you. 
+                        Rank options questions are question where the students rank options that are created by you.
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

File: src/main/webapp/instructorHelp.jsp [hunk 7]
```diff
@@ -7913,16 +7913,16 @@
...
                         <br>
-                    
-                        <br> To setup the question, enter the main question text, and add the options for the students to rank. You can configure if students can give the same rank multiple times. 
+
+                        <br> To setup the question, enter the main question text, and add the options for the students to rank. You can configure if students can give the same rank multiple times.
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

File: src/main/webapp/instructorHelp.jsp [hunk 8]
```diff
@@ -8123,7 +8123,7 @@
                                                 </ul>
                                             </div>
                                             <div class="feedback-path-others margin-top-7px" style="">
-                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">  
+                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

File: src/main/webapp/instructorHelp.jsp [hunk 9]
```diff
@@ -8351,17 +8351,17 @@
...
+
 
                 </div>
                 <div class="row">
                     <div class="col-sm-12">
                         <a name="fbRankRecipients">
                             <h4>Rank Recipients question</h4>
                         </a>
-                        Rank recipients questions are questions where the students are to rank students, teams, or instructors. 
+                        Rank recipients questions are questions where the students are to rank students, teams, or instructors.
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

File: src/main/webapp/instructorHelp.jsp [hunk 10]
```diff
@@ -8351,17 +8351,17 @@
...
                         <br>
-                        <br> The options to rank are determined by the feedback path selected for the question. You can configure if students can give the same rank multiple times. 
+                        <br> The options to rank are determined by the feedback path selected for the question. You can configure if students can give the same rank multiple times.
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

File: src/main/webapp/instructorHelp.jsp [hunk 11]
```diff
@@ -8504,7 +8504,7 @@
                                                 </ul>
                                             </div>
                                             <div class="feedback-path-others margin-top-7px" style="">
-                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">  
+                                                <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

File: src/main/webapp/instructorHelp.jsp [hunk 12]
```diff
@@ -8730,10 +8730,10 @@
...
                                 <input type="hidden" name="showrecipientto" value="RECEIVER,INSTRUCTORS">
                                 <input type="hidden" name="user" value="inst@email.com">
-                            </form>  
+                            </form>
                         </div>
-                        
-                        <br> The statistics for both rank questions show the average rank an option/recipient received. Ties are handled during the computation of statistics. If duplicate ranks are allowed to be given, ties are resolved by assigning the best rank to the occurences of tied values. For example, for the data {1, 3, 3, 4}, the ranks will be converted to {1, 2, 2, 4}. 
+
+                        <br> The statistics for both rank questions show the average rank an option/recipient received. Ties are handled during the computation of statistics. If duplicate ranks are allowed to be given, ties are resolved by assigning the best rank to the occurences of tied values. For example, for the data {1, 3, 3, 4}, the ranks will be converted to {1, 2, 2, 4}.
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: here

**Comment by iamhunter** [comment]
Hmm. that's odd. I reverted the changes from the last time and tried to do it again whilst being meticulous about not taking out any white space, but GitHub says that I have taken out quite a few. Any idea why that would be the case?

**Comment by iamhunter** [comment]
Should I close this pull request and see if it works if I start fresh?

**Comment by samsontmr** [comment]
Are you using SourceTree? If you are, you can choose the lines that you want to commit.
And are you using Atom to edit the files? I had a similar problem with the whitespaces when I used Atom

File: src/main/webapp/instructorHelp.jsp [hunk 13]
```diff
@@ -229,7 +229,11 @@
                             Go to the ‘Home’ page, click on the ‘Students‘ button on the corresponding course, and choose ‘Enroll’
                         </li>
                         <li>
+<<<<<<< HEAD
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: This indicates a merge conflict

File: src/main/webapp/instructorHelp.jsp [hunk 14]
```diff
@@ -229,7 +229,11 @@
...
+                            Go to the ‘Courses’ page and click the ‘Enroll‘ button of the corresponding course
+=======
                             Go the the ‘Courses’ page and click the ‘Enroll‘ button of the corresponding course
+>>>>>>> parent of 075b7d1... Duplicate "the" in jsp files #6460
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: You will need to resolve it

File: src/main/webapp/instructorHelp.jsp [hunk 15]
```diff
@@ -229,7 +229,11 @@
                             Go to the ‘Home’ page, click on the ‘Students‘ button on the corresponding course, and choose ‘Enroll’
                         </li>
                         <li>
-                            Go the the ‘Courses’ page and click the ‘Enroll‘ button of the corresponding course
+<<<<<<< HEAD
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: Remove this line

File: src/main/webapp/instructorHelp.jsp [hunk 16]
```diff
@@ -229,7 +229,11 @@
...
+                            Go to the ‘Courses’ page and click the ‘Enroll‘ button of the corresponding course
+=======
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: and this

File: src/main/webapp/instructorHelp.jsp [hunk 17]
```diff
@@ -229,7 +229,11 @@
...
+                            Go to the ‘Courses’ page and click the ‘Enroll‘ button of the corresponding course
+>>>>>>> parent of 075b7d1... Duplicate "the" in jsp files #6460
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: to this

File: src/main/webapp/instructorHelp.jsp [hunk 18]
```diff
@@ -661,7 +665,7 @@
                         This may be a good policy to adopt (If students know this policy, they will be more honest in reporting contribution values):
                         <br>
                         <br>
-                        <span class="italic">It is OK to do less than an equal share of the the work, as long as you are willing to accept a lower grade than your team members. It is your right to decide how much time you allocate to this course. How you prioritize your work is your own business.</span>
+                        <span class="italic">It is OK to do less than an equal share of to the work, as long as you are willing to accept a lower grade than your team members. It is your right to decide how much time you allocate to this course. How you prioritize your work is your own business.</span>
```
Comment (samsontmr) [inline src/main/webapp/instructorHelp.jsp]: the "to" is not necessary

**Comment by iamhunter** [comment]
At this point, I've dug down the rabbit hole of screwing up pretty far and both my github and source tree are messed up so we're gonna close this pull request and then start over tomorrow with a clean slate and figure it out from there.  I appreciate your help but I think I'll be better off just doing it all in one fell swoop

**Comment by samsontmr** [comment]
@iamhunter Actually you were almost there

**Pull request closed without merging**